### PR TITLE
Fix up readmes

### PR DESCRIPTION
**Pull request by kevmoo**
Use ``` for examples - fixes https://github.com/dart-lang/stagehand/issues/577
Drop package name - fixes https://github.com/dart-lang/stagehand/issues/576

### CODE REVIEW

File: templates/package-simple/README.md [hunk 1]
```diff
@@ -9,11 +7,14 @@ Created from templates made available by Stagehand under a BSD-style
 
 A simple usage example:
 
-    import 'package:__projectName__/__projectName__.dart';
+```dart
+
```
Comment (kwalrath) [inline templates/package-simple/README.md]: Why do we have a blank line here? It doesn't seem to affect the output, but it seems unnecessary.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ## 3.1.3
 
+- All templates: small improvements to `README.md` generation.
```
Comment (kwalrath) [inline CHANGELOG.md]: Nit: It's not really to the `README.md` generation, but to the generated `README.md`.

File: templates/package-simple/README.md (hunk 1 above)
Comment (kevmoo) [reply templates/package-simple/README.md]: ack

File: CHANGELOG.md (hunk 2 above)
Comment (kevmoo) [reply CHANGELOG.md]: ack

**Comment by kwalrath** [comment]
Want me to release stagehand, or will you do it?

**Comment by kevmoo** [comment]
Go ahead. Not urgent, though.

**Comment by kwalrath** [comment]
I'll make a note to do it Monday.

**Comment by kevmoo** [comment]
k

**Pull request merged**